docs/victorialogs/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ according to [these docs](https://docs.victoriametrics.com/victorialogs/quicksta
 
 * FEATURE: [web UI](https://docs.victoriametrics.com/victorialogs/querying/#web-ui): add "Live" tab that allows monitoring logs in real-time as they arrive. This feature helps users to observe the most recent log entries without manual refreshing, making troubleshooting and monitoring more efficient. See [#7046](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7046).
 * FEATURE: [dashboards/cluster](https://grafana.com/grafana/dashboards/23274) and [dashboards/single](https://grafana.com/grafana/dashboards/22084): add panels for [Pressure Stall Information (PSI)](https://docs.kernel.org/accounting/psi.html) metrics to dashboards. They could help to identify shortage of resources for VictoriaLogs components.
+* FEATURE: [`math` pipe](https://docs.victoriametrics.com/victorialogs/logsql/#math-pipe): add `now()` function, which returns the current [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) in nanoseconds.
```

Comment:
@func25 , please add a link to the corresponding issue(s) in the changelog line next time (the https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9013 ) in this case). This simplifies the [release process](https://docs.victoriametrics.com/victoriametrics/release-guide/), which includes the step for adding the `the issue / feature request has been resolved in the release v1...` across all the linked issues, which are resolved / fixed in the published release. This also helps users obtaining more details about the corresponding change / bugfix mentioned in the changelog, by navigating to the linked issues.